### PR TITLE
Square methods

### LINKED ISSUE #59: Add fill_square() and stroke_square() to complement the square() method
AC:
 * [x] Add `fill_square` and `stroke_square` methods
    * Should be global immutable
 * [x] Add unit tests
 * [x] Add documentation

### PR DESCRIPTION
**Pull request by lawruble13**
Adds unit tests and documentation for square functions, closing #59.

### CODE REVIEW

File: docs/shapes_text.md [hunk 1]
```diff
@@ -107,6 +107,108 @@ Results in:
...
 
+### Squares
+
+To create a square there are 3 options:
+
+| Function                                 | Description                                        |
+|------------------------------------------|----------------------------------------------------|
+| [square(x, y, s)](#square)               | Draw a square filled with color, and with a stroke |
+| [fill_square(x, y, s)](#fill_square)     | Draw an square filled with a color                 |
+| [stroke_square(x, y, s)](#stroke_square) | Draw the outline of a square (it's stroke)         |
```
Comment (AlphaRLee) [inline docs/shapes_text.md]: Grammar typo on "its"
```suggestion
| [stroke_square(x, y, s)](#stroke_square) | Draw the outline of a square (its stroke)         |
```

**Review by Descent098 — APPROVED** [review]
Confirmed functionality and docs look good. Approving

**Review by AlphaRLee — APPROVED** [review]
Looks good to me. After you fix a minor grammar error then it should be ready to merge in

**Review by raduschirliu — APPROVED** [review]
Looks good, nice!

**Pull request merged**